packages/edit-site/src/components/page-library/grid-item.js

```diff
@@ -128,7 +129,14 @@ export default function GridItem( { categoryId, composite, icon, item } ) {
 						className="edit-site-library__pattern-title"
 					>
 						{ icon && <Icon icon={ icon } /> }
-						<Heading level={ 5 }>{ item.title }</Heading>
+						<Heading level={ 5 }>
+							<Button
+								variant="link"
```

Comment:
The button needs to have `tabIndex="-1"`, otherwise this extra focusable element breaks the roving tab index that `Composite`/`CompositeItem` use.

Reply:
Ah right 👍

Reply:
✅ I fixed this and it seems to restore the tab index.

Reply:
Thanks ❤️